build.gradle

```diff
@@ -4,7 +4,7 @@ subprojects {
   apply plugin: "signing"
 
   group = "com.marklogic"
-  version = "1.3-SNAPSHOT"
+  version = "1.3.0"
```

Comment:
Can you make this 1.4-SNAPSHOT? I think the Jenkins pipeline will throw an error if it doesn't have "SNAPSHOT" in it.